.github/workflows/test-report.yaml

```diff
@@ -43,5 +43,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@4.1.7
         with:
+          repository-name: dask/distributed
```

Comment:
IIUC, this would try pushing to dask/distributed from user forks

> Allows you to specify a different repository path so long as you have permissions to push to it. This should be formatted like so: JamesIves/github-pages-deploy-action. You'll need to use a PAT in the token input for this configuration option to work properly.

I think what we need is an if clause similar to https://github.com/dask/distributed/blob/2fffe74dde40238a3387d12740ff9a54c16acb9f/.github/workflows/tests.yaml#L18

Reply:
Thanks for catching that Ian! 😄 

Submitted a fix in PR ( https://github.com/dask/distributed/pull/5959 )